### PR TITLE
feat: add scroll + fix typos

### PR DESCRIPTION
**Pull request by omahs**
feat: add scroll + fix typos

### CODE REVIEW

**Review by joshstevens19 — APPROVED** [review]

**Comment by joshstevens19** [comment]
Will deploy tomorrow thanks for PRA

**Comment by joshstevens19** [comment]
https://github.com/joshstevens19/ethereum-multicall/releases/tag/2.19.0

File: README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ The below networks are supported by default, and custom networks can be supporte
 | Etherlite               | 111        |
 | Linea Testnet           | 59140      |
 | Linea                   | 59144      |
+| Scroll                  | 534352     |
```
Comment (Turupawn) [inline README.md:69]: Scroll mainnet is not yet launched. This should be renamed to `Scroll Alpha`

File: src/multicall.ts [hunk 2]
```diff
@@ -596,6 +596,7 @@ export class Multicall {
       case Networks.kcc:
       case Networks.lineaTestnet:
       case Networks.linea:
+      case Networks.scroll:
```
Comment (Turupawn) [inline src/multicall.ts:599]: Scroll Alpha multicall deployment is at `0x2117f703867a2B7E6813c7e5Edd96bf9a8d8eC30`

File: src/enums/networks.ts [hunk 3]
```diff
@@ -50,4 +50,5 @@ export enum Networks {
   etherlite = 111,
   lineaTestnet = 59140,
   linea = 59144,
+  scroll = 534352,
```
Comment (Turupawn) [inline src/enums/networks.ts:53]: This should be renamed to scrollAlpha

File: src/multicall.ts (hunk 2 above)
Comment (Turupawn) [reply src/multicall.ts:599]: And also should be renamed to Networks.scrollAlpha

**Review by Turupawn — COMMENTED** [review]
Renaming from Scroll to Scroll Alpha

**Comment by joshstevens19** [comment]
@Turupawn please fix with a PR and we will merge and deploy

**Pull request merged**